systemd/systemd.go

```diff
@@ -1416,33 +1422,13 @@ X-SnapdOrigin={{.}}
 {{- end}}
 `
 
-// We want kernel-modules components to be mounted before modules are
-// loaded (that is before systemd-{udevd,modules-load}).
-const beforeDriversLoadUnitTmpl = `[Unit]
-Description={{.Description}}
-DefaultDependencies=no
-After=systemd-remount-fs.service
-Before=sysinit.target
-Before=systemd-udevd.service systemd-modules-load.service
-Before=umount.target
-Conflicts=umount.target
-
-[Mount]
-What={{.What}}
-Where={{.Where}}
-Type={{.Fstype}}
-Options={{join .Options ","}}
-
-[Install]
-WantedBy=sysinit.target
-{{- with .Origin}}
-X-SnapdOrigin={{.}}
-{{- end}}
-`
+func IsBeforeDriversLoadMountUnit(mType MountUnitType) bool {
```

Comment:
do we need to export this? if we do it needs a comment

Reply:
Not really, for some reason I thought that maybe the  go template would need it exported, but is not the case. Unexported now.